Cargo.toml

```diff
@@ -1,8 +1,9 @@
 [workspace]
 members = [ 
+    "crates/cli", 
```

Comment:
i wonder if we should name it web5_cli to keep the theme going, and then maybe one day there will be a tbdex_cli

Reply:
agreed, done